### PR TITLE
docs(website): add multi repo design proposal

### PR DESCRIPTION
**Pull request by kosmoz**
Ref #195 

## ✅ Checks
- [x] My code requires changes to the documentation
- [x] I have updated the documentation as required

## ℹ Additional Information
Link to the design proposal: https://github.com/glasskube/glasskube/blob/kosmoz/website/repositories-design/website/docs/04_design/repositories.md

### CODE REVIEW

**Review by christophenne — APPROVED** [review]
LGTM!

File: website/docs/04_design/repositories.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  The repositories each package is available from are shown in a new column.
+  If a package is installed and it is available from more than one repository, the one used in the installation is marked in this column with "(used)"
+  If a package is not installed and it is available from more than one repository, the latest version shown is the highes one available across repositories.
+- `glasskube describe` shows the repositories the package is available from and the one that is used for installation.
+
+### UI Design
+
+#### Repository Management
+
+Mockup of the new settings page where the packare repository configuration will be situated.
```
Comment (pmig) [inline website/docs/04_design/repositories.md:114]:
```suggestion
Mockup of the new settings page where the package repository configuration will be situated.
```

**Review by pmig — COMMENTED** [review]
Looks good to me. really nice job

File: website/docs/04_design/repositories.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+On the list page, packages from all installed repositories are shown.
+If two or more repositories feature a package with the same name and that package is not installed, it is only shown once, where the iconUrl and shortDescription are taken from the first repository ordered alphabetically, that is not the default repository.
+
+On the details page, if the package is not installed and featured in two or more repositories, there is a dropdown at the top of the page where a user can select the repository they want to install the package from.
+When changing the dropdown value, the page is refreshed with data from the new selected repository.
+All data shown comes from the selected repository.
+
+If a package is already installed or only available from one repository, the dropdown is not shown.
+
+### Migration & Compatibility
```
[the remark this answers is not in the document]
Comment (kosmoz) [reply website/docs/04_design/repositories.md:137]: If the bootstrap command doesn't create a default repository there is no default repository…

File: website/docs/04_design/repositories.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+The `bootstrap` command creates a default repository in the cluster by default.
+This repository has the name "glasskube" and repositoryUrl "https://packages.dl.glasskube.dev/packages/".
```
[the remark this answers is not in the document]
Comment (kosmoz) [reply website/docs/04_design/repositories.md:140]: but why?

File: website/docs/04_design/repositories.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+apiVersion: packages.glasskube.dev/v1alpha1
+kind: Package
+metadata:
+  name: package-sample
+spec:
+  packageInfo:
+    name: foo
+    version: v1.0.0+1
+    # repositoryName is optional
+    repositoryName: private-repo
```
[the remark this answers is not in the document]
Comment (kosmoz) [reply website/docs/04_design/repositories.md:68]: Yes, I also think this should be part of the `PackageRepository` spec

**Review by pmig — APPROVED** [review]

**Pull request merged**